dashboards/AdminRouter/AdminRouter-Details-Upstreams.json

```diff
@@ -0,0 +1,4541 @@
+      },
+      "lines": true,
+      "linewidth": 1,
+      "links": [],
+      "nullPointMode": "connected",
+      "percentage": false,
+      "pointradius": 1,
+      "points": false,
+      "renderer": "flot",
+      "seriesOverrides": [],
+      "spaceLength": 10,
+      "stack": true,
+      "steppedLine": false,
+      "targets": [
+        {
+          "expr": "max((rate(nginx_vts_server_requests_total{host=\"*\",code=\"5xx\",dcos_component_name=\"Admin Router\",instance=~\"$master_prometheus\"}[$rate_interval]) / ignoring(code) rate(nginx_vts_server_requests_total{host=\"*\",code=\"total\",dcos_component_name=\"Admin Router\",instance=~\"$master_prometheus\"}[$rate_interval])) * 100)",
+          "format": "time_series",
+          "intervalFactor": 1,
+          "legendFormat": "5xx",
+          "refId": "D"
+        },
+        {
+          "expr": "max((rate(nginx_vts_server_requests_total{host=\"*\",code=\"4xx\",dcos_component_name=\"Admin Router\",instance=~\"$master_prometheus\"}[$rate_interval]) / ignoring(code) rate(nginx_vts_server_requests_total{host=\"*\",code=\"total\",dcos_component_name=\"Admin Router\",instance=~\"$master_prometheus\"}[$rate_interval])) * 100)",
+          "format": "time_series",
+          "intervalFactor": 1,
+          "legendFormat": "4xx",
+          "refId": "C"
+        },
+        {
+          "expr": "max((rate(nginx_vts_server_requests_total{host=\"*\",code=\"3xx\",dcos_component_name=\"Admin Router\",instance=~\"$master_prometheus\"}[$rate_interval]) / ignoring(code) rate(nginx_vts_server_requests_total{host=\"*\",code=\"total\",dcos_component_name=\"Admin Router\",instance=~\"$master_prometheus\"}[$rate_interval])) * 100)",
+          "format": "time_series",
+          "intervalFactor": 1,
+          "legendFormat": "3xx",
+          "refId": "B"
+        },
+        {
+          "expr": "max((rate(nginx_vts_server_requests_total{host=\"*\",code=\"2xx\",dcos_component_name=\"Admin Router\",instance=~\"$master_prometheus\"}[$rate_interval]) / ignoring(code) rate(nginx_vts_server_requests_total{host=\"*\",code=\"total\",dcos_component_name=\"Admin Router\",instance=~\"$master_prometheus\"}[$rate_interval])) * 100)",
+          "format": "time_series",
+          "interval": "",
+          "intervalFactor": 1,
+          "legendFormat": "2xx",
+          "refId": "A"
+        }
+      ],
+      "thresholds": [],
+      "timeFrom": null,
+      "timeShift": null,
+      "title": "Admin Router Nginx Status Percentage: $master_prometheus",
+      "tooltip": {
+        "msResolution": true,
+        "shared": true,
+        "sort": 0,
+        "value_type": "individual"
+      },
+      "type": "graph",
+      "xaxis": {
+        "buckets": null,
+        "mode": "time",
+        "name": null,
+        "show": true,
+        "values": []
+      },
+      "yaxes": [
+        {
+          "decimals": 1,
+          "format": "percent",
+          "label": "log(2) % HTTP Errors",
```

Comment:
What is the `(2)` here? The logbase is 10, but for percentages, I don't think the base matters much, since it is labelled. Maybe just `log % HTTP Errors`, unless `(2)` means something else.

Reply:
`log % HTTP status` it is ;)